### PR TITLE
Fix sekaikomik

### LINKED ISSUE #8892: Sekaikomik: Blocked by Cloudflare
### Source information

Sekaikomik 1.2.19

### Steps to reproduce

1. open
2. eror 403
3.  open in browser
4. blocked by cloudflare

47344e2c-66fd-4a21-af8c-800c81ee2cd2

### Expected behavior

open normally

### Actual behavior

blocked by cloudflare

### Tachiyomi version

0.12.1

### Android version

Android 11

### Other details

blocked by cloudflare

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Claudemirovsky**
Closes #8892

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Annotated `Source` or `SourceFactory` classes with `@Nsfw` when appropriate
- [x] Added the `containsNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `versionId` if a source's name or language were changed

### CODE REVIEW

File: multisrc/overrides/wpmangareader/sekaikomik/src/SekaiKomik.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import java.util.Locale
+@Nsfw
+class Sekaikomik : WPMangaReader("Sekaikomik", "https://www.sekaikomik.biz", "id", dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale("id"))) {
+    override val client: OkHttpClient = network.cloudflareClient.newBuilder()
+        // thanks https://stackoverflow.com/a/53233345
+        .addNetworkInterceptor { chain ->
+            chain.proceed(
+                chain.request().newBuilder()
+                    // Anti error 403
+                    .header("User-Agent", USER_AGENT)
```
Comment (alessandrojean) [inline multisrc/overrides/wpmangareader/sekaikomik/src/SekaiKomik.kt:16]: You can add the headers by overriding the `headersBuilder()` method in the class.

**Review by alessandrojean — CHANGES_REQUESTED** [review]

File: multisrc/overrides/wpmangareader/sekaikomik/src/SekaiKomik.kt (hunk 1 above)
Comment (Claudemirovsky) [reply multisrc/overrides/wpmangareader/sekaikomik/src/SekaiKomik.kt:16]: I tried this now, but for some strange reason it is only working in the webview, the headers are not being used in the reader.... :(

Comment (alessandrojean) [reply multisrc/overrides/wpmangareader/sekaikomik/src/SekaiKomik.kt:16]: Weird, they're supposed to be used in any request the source client does. You can check if they're in fact being used by checking the logcat and filtering by the`OkHttpClient` tag.

Comment (Claudemirovsky) [reply multisrc/overrides/wpmangareader/sekaikomik/src/SekaiKomik.kt:16]: I was using `HttpCanary` to see if the headers were being sent, and both it and logcat show that they are not, and I don't see anything wrong in the code, I even compared it with the code from another source in wpmangareader([ngokomik](https://github.com/tachiyomiorg/tachiyomi-extensions/blob/master/multisrc/overrides/wpmangareader/ngomik/src/Ngomik.kt)).
my poor brain is melting, it's like it doesn't want to work on this source

**Pull request closed without merging**